src/2_RTB/NormeDiProgetto/content.typ

```diff
@@ -186,3 +186,79 @@ Potrebbero esservi delle issue aperte all'interno dell'_ITS_ che non hanno un co
 Per poter prendere una qualsiasi decisione è necessario vi siano due condizioni:
 + Si deve raggiungere il #glossary[quorum] di quattro persone su sei;
 + La decisione deve essere verbalizzata e motivata.
+
+=== Ruoli Progetto
+
+Durante il periodo di sviluppo del progetto, ogni membro assumerà 6 ruoli distinti con compiti diversificati, al fine di garantire una gestione completa ed efficace delle diverse fasi e aspetti del lavoro. Questi ruoli contribuiranno a promuovere la collaborazione sinergica e l'ottimizzazione delle risorse all'interno del team. 
+
+I ruoli assunti sono i seguenti:
+
+==== Responsabile  
+
+        Figura chiave nell'ambito della gestione di progetti. Ha la responsabilità sul controllo, coordinamento e relazioni esterne.
+        I suoi compiti sono:
+
+         - Gestione della comunicazione con la Proponente (si fa uso della piattaforma Element);
+         - Preparazione dell'ordine del giorno per la successiva riunione, anche sulla base
+           dei punti individuati dagli altri componenti;
+         - Redazione dei verbali interni ed esterni;
+         - Stesura e e avanzamento del documento “Piano di progetto”;
+         - Creazione dei diari di bordo;
+         - Scaricamento dei verbali esterni convalidati e immagazzinamento dentro la cartella IncontriAziende/VerbaliEsterni_Convalidati su Google Drive.
```

Comment:
Non serve inserire il nome esatto della cartella dove salviamo i verbali esterni, direi semplicemente "Upload dei verbali esterni convalidati tramite firma dalla Proponente nella cartella apposita in Google Drive."